shared/teams/team/rows/member-row/index.tsx

```diff
@@ -66,63 +70,153 @@ export const TeamMemberRow = (props: Props) => {
+    )
+
+    const menuHeader = {
+      title: 'header',
+      view: (
+        <MenuHeader
+          username={props.username}
+          fullName={props.fullName}
+          label={
+            <Kb.Box2 direction="horizontal">
+              <Kb.Text type="BodySmall">{crown}</Kb.Text>
+              <Kb.Text type="BodySmall">{roleLabel}</Kb.Text>
+            </Kb.Box2>
+          }
+        />
+      ),
+    }
+
+    const menuItems: Kb.MenuItems = [
+      'Divider',
+      ...(props.youCanManageMembers
+        ? ([
+            {icon: 'iconfont-chat', onClick: props.onChat, title: 'Add to channels...'},
+            {icon: 'iconfont-crown-admin', onClick: props.onClick, title: 'Edit role...'},
+          ] as Kb.MenuItems)
+        : []),
+      {icon: 'iconfont-person', onClick: props.onOpenProfile, title: 'View profile'},
+      {icon: 'iconfont-chat', onClick: props.onChat, title: 'Chat'},
+      ...(props.youCanManageMembers || !isYou ? (['Divider'] as Kb.MenuItems) : []),
+      ...(props.youCanManageMembers
+        ? ([
+            {
+              danger: true,
+              icon: 'iconfont-remove',
+              onClick: props.onRemoveFromTeam,
+              title: 'Remove from team',
+            },
+          ] as Kb.MenuItems)
+        : []),
+      ...(!isYou
+        ? ([
+            {
+              danger: true,
+              icon: 'iconfont-block',
+              onClick: props.onBlock,
+              title: 'Block',
+            },
+          ] as Kb.MenuItems)
+        : []),
+    ]
+    const {showingPopup, toggleShowingPopup, popupAnchor, popup} = Kb.usePopup(attachTo => (
+      <Kb.FloatingMenu
+        header={menuHeader}
+        attachTo={attachTo}
+        closeOnSelect={true}
+        items={menuItems}
+        onHidden={toggleShowingPopup}
+        visible={showingPopup}
+      />
+    ))
+
+    const actions = (
+      <Kb.Box2 direction="horizontal" gap="tiny" style={styles.mobileMarginsHack}>
+        {popup}
+        <Kb.Button
+          icon="iconfont-chat"
+          iconColor={Styles.globalColors.black_50}
+          mode="Secondary"
+          onClick={props.onChat}
+          small={true}
+          tooltip="Open chat"
+        />
+        <Kb.Button
+          icon="iconfont-ellipsis"
+          iconColor={Styles.globalColors.black_50}
+          mode="Secondary"
+          onClick={toggleShowingPopup}
+          ref={popupAnchor}
+          small={true}
+          tooltip="More actions"
+        />
+      </Kb.Box2>
+    )
+
+    return (
+      <Kb.ListItem2
+        action={actions}
+        onlyShowActionOnHover="fade"
+        height={Styles.isMobile ? 90 : 64}
+        icon={checkCircle}
+        iconStyleOverride={styles.checkCircle}
+        containerStyleOverride={styles.listItemMargin}
+        type="Large"
+        body={body}
+        firstItem={isOwner}
+        style={selected ? styles.selected : undefined}
+        onClick={isOwner ? undefined : () => onSelect(!selected)}
```

Comment:
Also, how do we navigate to the member page? Maybe clicking the item body goes to the member page unless someone is already selected, then it toggles the selection?